src/tests/agenda/test_agenda_schedule_export.py

```diff
@@ -115,7 +115,7 @@ def test_schedule_frab_json_export(
             follow=True,
         )
     client.force_login(orga_user)
-    with django_assert_max_num_queries(21):
```

Comment:
I'm not wild on the increased database queries. What's causing them – the URL lookup or the track lookup?

Reply:
probably the tracks lookup, they were not part of this export before this change